### DISCUSSION TITLE
Sloeber IDE and RP2040 USB bulk device

### DISCUSSION THREAD

**Discussion by qwarkx** [Sloeber/arduino-eclipse-plugin#1484] [General]
Hi everyone,

I been tiring to get the rp2040 running as a USB bulk device with which I can send bulk device packets to the PC, achieving high data throughput, using the windows available USB driver. 

If you can give me guidance in regards of this, that would be really helpful. 
- how to set up a custom usb device
- how i use dma for the packet which is sent through the USB
- how to access a variable by bout cores of the rp2040 because one core is gathering data and the second core is sending data through USB

**Comment by jantje** [comment]
I guess you are are asking the right questions at the wrong place.
What I mean : I do not see how any of these questions relate to Sloeber
